src/Domain.LinnApps/Requisitions/CreationStrategies/LoanOutCreationStrategy.cs

```diff
@@ -29,7 +29,16 @@ public Task<RequisitionHeader> Create(RequisitionCreationContext context)
                 throw new CreateRequisitionException("Loan Out function requires a Loan document number");
             }
 
-            return this.requisitionManager.CreateLoanReq(context.Document1Number.Value);
+            var req = await this.requisitionManager.CreateLoanReq(context.Document1Number.Value);
+            
+            // need to add some extra fields to make picking and booking possible
+            // REQ_UT does this from function code WVI but we know what they should be
+            req.FromState = "STORES";
+            req.ToState = "STORES";
+            req.ToCategory = "FREE";
+            req.ReqSource = "STORES2";
```

Comment:
I've generally been trying to make the fields setters protected when there's some benefit to doing so (e.g. should these things be fixed across the lifetime of a req? if so you could protect them from public setters and then set from inside the in the constructor or something.  That way you get the constraint that 'these things can never change' for free via encapsulation. Feel free to ignore if you think this is more trouble than its worth, just thought I'd mention it since thats the general convention I've been following